content/en/providers/check-in/sp/_index.md

```diff
@@ -66,30 +66,30 @@ more information.
 The integration follows a two-step process:
 
 1. Register your Service Provider and test integration with the **demo**
-   instance of EGI Check-in by selecting the "Demo" integration
-   environment during registration through the EGI Federation Registry Portal.
-   The demo instance allows for testing authentication and authorisation
-   through the academic and social Identity Providers connected to Check-in
-   without affecting the production Check-in service. Note that while the demo
-   instance has identical functionality to the production instance, no
-   information is shared between the two systems.
+   instance of EGI Check-in by selecting the "Demo" integration environment
+   during registration through the EGI Federation Registry Portal. The demo
+   instance allows for testing authentication and authorisation through the
+   academic and social Identity Providers connected to Check-in without
+   affecting the production Check-in service. Note that while the demo instance
+   has identical functionality to the production instance, no information is
+   shared between the two systems.
    - You can also test new features of Check-in that are not available in
      production yet, by registering your Service Provider and testing
      integration with the **development** instance of Check-in. In the
      development instance service requests can be self-reviewed without the need
-     to wait for aproval from an administrator. As with the demo instance, the
+     to wait for approval from an administrator. As with the demo instance, the
      development instance allows for testing authentication and authorisation
      without affecting the production Check-in service. **NB: the list of
      supported Identity Providers in the development instance is limited.
-     Therefore, we recommend using the [EGI SSO](https://sso.egi.eu/admin/)
-     or any of the social identity providers to test the login workflow when
-     using the development instance.**
+     Therefore, we recommend using the [EGI SSO](https://sso.egi.eu/admin/) or
+     any of the social identity providers to test the login workflow when using
```

Comment:
I would revert the order: using social accounts or the EGI SSO. We don't really want more people to create accounts in the EGI SSO unless they have no other choice.

Reply:
Resolved by commit [Fix typos](https://github.com/EGI-Federation/documentation/pull/332/commits/7c9c7855873f8858ed9f14a4a5119ff667c32741)